### ISSUE TITLE
Harness failing to bind contact-summary context onto forms containing elements where name is non-self-closing HTML tag (eg. form)

### ISSUE THREAD

**Issue by jonathanbataire** [medic/cht-conf-test-harness#228]
# Description
Some config-ug-vht  tests are failing because forms logic relies on contact summary values passed into the forms.
Harness doesnt seem to pass these values into the form hence the form workflow logic fails then the test
### Example of failing tests
https://github.com/medic/config-ug-vht/blob/c4ba8c280c6545e14fd7b5c999413bc23b085cf9/test/contact-summary/family_planning_card.spec.js#L78

Tested using v3.0.6 branch

**Comment by kennsippell** [comment]
The contact summary isn't being bound onto the form because [this condition](https://github.com/medic/cht-core/blob/enketo-service-refactor/webapp/src/js/enketo/enketo-form-manager.js#L94) is failing to evaluate.  The contact information isn't properly being set on the `content`.

**Comment by jkuester** [comment]
Short answer is no, I have not seen this before. 

That being said, I am really confused by everything in this thread.  (Maybe I just need more coffee....)

- The `fp_registration` form that is used in the `family_planning_card` tests does not seem to have any logic in it that references `contact-summary`  (either in the version from the linked commit or in master).
- I cannot recreate your described jquery behavior. (Both examples return `1` for me when just run in isolation.)
- I tried spinning up the `fp_registration` form on a CHT instance running the `enketo-service-refactor` and it seemed to work fine

**Comment by jonathanbataire** [comment]
@jkuester it's the fp_follow_up form
That's where you get the stuff
The fp_registration will trigger the follow up task

**Comment by jkuester** [comment]
Ah of course!  :facepalm:   

@jonathanbataire do you have a branch where the test is failing?

**Comment by jonathanbataire** [comment]
> Ah of course! facepalm
> 
> @jonathanbataire do you have a branch where the test is failing?

This test https://github.com/medic/config-ug-vht/blob/14cdcc0d54893d06e81b370ece723557b027bad7/test/contact-summary/family_planning_card.spec.js#L78

**Comment by jkuester** [comment]
Found it!  

So, the longer I looked at this the more familiar it felt.  Finally I realized it is because of a change in how JQuery deals with self-closing xml tags: https://jquery.com/upgrade-guide/3.5/.

Basically, tags like `p`, `span` and `form` are  not allowed to be self-closing in HTML. _But_ they can be in XML.  However, JQuery uses the "htmlPrefilter" even when parsing XML.  In the case of the `fp_registration` form, one of the fields in the model is `<form/>`.  

The solution here is just to update the test-harness to implement the override suggested in the above linked jQuery upgrade guide (similar to what we did in cht-core: https://github.com/medic/cht-core/blob/722d4802da6c5c2f9a7f0f2a840446bd39e94d6d/webapp/src/ts/main.ts#L45
